src/providers/ipa/ipa_subdomains_server.c

```diff
@@ -697,7 +697,7 @@ ipa_ctx_new(struct be_ctx *be_ctx,
     }
 
     ret = domain_to_basedn(be_ctx,
-                           dp_opt_get_string(ipa_options->id->basic, IPA_KRB5_REALM),
+                           dp_opt_get_string(ipa_options->id->basic, SDAP_KRB5_REALM),
                            &basedn);
```

Comment:
Hi,

I think it would be more straight forward to use `dp_opt_get_string(ipa_options->basic, IPA_KRB5_REALM)` or maybe even better `dp_opt_get_string(ipa_options->basic, IPA_DOMAIN)` because the base DN is typically derived from the domain name and in theory the Kerberos realm might differ more than just being upper-case.

bye,
Sumit

Reply:
Thank you, fixed and PR updated.

Reply:
Thanks.